src/main/context.ts

```diff
@@ -319,9 +326,9 @@ export async function getEventDefaults(release?: string): Promise<Event> {
   // The event defaults are cached as long as the app is running. We create the
   // promise here synchronously to avoid multiple events computing them at the
   // same time.
-  if (!defaultsPromise) {
-    defaultsPromise = _getEventDefaults(release);
+  if (!cachedDefaultsPromise) {
+    cachedDefaultsPromise = _getEventDefaults(release);
   }
 
-  return await defaultsPromise;
+  return await cachedDefaultsPromise;
```

Comment:
I wonder if we should cache the actual value instead of having to resolve the promise every time.